notebooks/MAST/K2/K2_Lightcurve/k2_lightcurve.ipynb

```diff
@@ -0,0 +1,334 @@
+    "ax.set_xlim(2920., 2950.)\n",
+    "ax.set_ylim(0.1e7, 0.2e7)\n",
+    "\n",
+    "# Let's label the axes and define a title for the figure.\n",
+    "fig.suptitle(\"TRAPPIST-1 Light Curve - Campaign 12\")\n",
+    "ax.set_ylabel(\"PDCSAP Flux (e-/s)\")\n",
+    "ax.set_xlabel(\"Time (TBJD)\")\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Intringuingly, some of the largest outliers in the positive flux direction are flagged: are these bad measurements that should be excised from the time series?  Finding out the quality flag value and converting the value to its consitutent bit masks to understand why these points are flagged would be the first step.  We encourage you to do this as a follow-up to this tutorial."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Displaying The Aperture Pixel Information\n",
+    "\n",
+    "Let's read in the second FITS extension HDU to display the aperture information.  First, let's read in the aperture pixels from the HDU."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with fits.open(fits_file, mode=\"readonly\") as hdulist:\n",
+    "    aperture = hdulist[2].data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's plot the pixels as an image."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Start figure and axis.\n",
+    "fig, ax = plt.subplots()\n",
+    "\n",
+    "# Display the pixels as an image.\n",
+    "cax = ax.imshow(aperture, cmap=plt.cm.YlGnBu_r, origin=\"lower\")\n",
+    "\n",
+    "# Add a color bar.\n",
+    "cbar = fig.colorbar(cax)\n",
+    "\n",
+    "# Add a title to the plot.\n",
+    "fig.suptitle(\"TRAPPIST-1 Aperture - Campaign 12\")\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Understanding The Aperture Pixel Values\n",
+    "\n",
+    "We see the pixel values are integers, but what do they mean?  The pixels are bitmasks that encode information about each pixel.  You can find a summary of what the different values mean [here](https://archive.stsci.edu/kepler/manuals/archive_manual.pdf#page=20).  For example, a pixel in the aperture that might have a value of 15 can be broken down into power of 2 like: 8+4+2+1 = 15.  Referencing the table of values, this means this particular pixel was in was used to calculate the Pixel Response Function (PRF) centroid, was used to calculate the flux weighted centroid, was part of the optimal photometric aperture, and was collected by the spacecraft.  Numpy has a bulit-in function that can convert an integer into a binary bit mask.  Let's use that now one one of the common values we see in our displayed image above."
```

Comment:
"this particular pixel was in was used to calculate the Pixel" -> extra "was in"

"one one" -> "on one"

Reply:
typos fixed.